media.go

```diff
@@ -17,6 +17,35 @@ import (
 	"time"
 )
 
+// StoryReelMention represent story reel mention
+type StoryReelMention struct {
+	X        float64 `json:"x"`
+	Y        float64 `json:"y"`
+	Z        int     `json:"z"`
+	Width    float64 `json:"width"`
+	Height   float64 `json:"height"`
+	Rotation float64 `json:"rotation"`
+	IsPinned int     `json:"is_pinned"`
+	IsHidden int     `json:"is_hidden"`
+	User     User
+}
+
+// StoryCTA represent story cta
```

Comment:
Can you add more information about CTA in the comments above of the CTA struct?
Maybe some users don't know CTA.